common/config/rush/command-line.json

```diff
@@ -0,0 +1,278 @@
+/**

```

Comment:
Sorry I accidentally posted this to another PR, not sure what happened, and I know this is already merged but I'm still curious about my question below.

Is there any hard rules about these files going into a folder called common? I ask mostly as for some of the other languages we have been putting similar engineering system type files under an "eng" directory.